assets/training/model_evaluation/src/compute_metrics.py

```diff
@@ -207,6 +207,10 @@ def filter_ground_truths(data, task_type, column_name=None):
     Returns:
         _type_: _description_
     """
+    if task_type in [constants.TASK.IMAGE_INSTANCE_SEGMENTATION, constants.TASK.IMAGE_OBJECT_DETECTION]:
+        # do not filter as these contains multiple required columns
+        return data
```

Comment:
what is the format of the data expected here @rjaincc

Reply:
Apart from label column, we need image_metadata information as well, hence, we don't need filtering for OD/IS. 
@MadhuM02 - Please feel free to add further details.